src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -42,11 +42,20 @@
 		$field->upsell_options = array();
 
 		foreach ( $field->options as $key => $option ) {
+
 			$option->disabled = isset( $option->disabled ) ? $option->disabled : null;
 			$option->upsell   = isset( $option->upsell ) ? $option->upsell : false;
 
 			$option->is_selected = false;
 
+			// Ensure the "(do not override)" status is set up for Eventbrite imports, and "Published" is removed.
+			if ( 'eventbrite' === $option->id ) {
+
+				$post_statuses = array( 'do_not_override' => esc_html__( '(do not override)', 'the-events-calendar' ) ) + $post_statuses;
```

Comment:
I've fixed the one-liner, but I'm using `+` here because we want this status to be the first item in the array. Neither `array_merge` nor `array_unshift` quite fit what we want, so we use the `+`.

It is used in some other locations in the EA code as well for the same purpose.